gammapy/maps/axes.py

```diff
@@ -1982,7 +2017,7 @@ def pix_to_idx(self, pix, clip=False):
     @property
     def center(self):
         """Return `~astropy.time.Time` at interval centers."""
-        return self.time_mid
+        return self.edges_min + 0.5 * self.time_delta
```

Comment:
this does not return a time. So either change docstring or implementation.

Reply:
Note that the docstring of `time_delta` is incorrect too. the difference of `edges` will return a `Quantity` not a `TimeDelta`.

Reply:
I changed the docstring, I think returning a Quantity is the consistent solution, with `edges_min`, `edges_max`, `center` and the corresponding time based attributes `time_min`, `time_max`, `time_mid` (maybe we should rename to `time_center`?)

Reply:
For `time_delta` I changed to the same pattern: `time_delta` returns a `TimeDelta`, while `bin_width` returns a `Quantity`. This way we are consistent with the standard `MapAxis`, but get the additional `.time_` attributes for convenience.